libs/s.js

```diff
@@ -5,7 +5,7 @@
 (function () {
 
   function errMsg(errCode, msg) {
-    return (msg || "") + " (SystemJS https://github.com/systemjs/systemjs/blob/master/docs/errors.md#" + errCode + ")";
+    return (msg || "") + " (SystemJS https://git.io/JvFET#" + errCode + ")";
```

Comment:
git.io links are not working anymore, we had a PR for that, let's not check this in